.gitlab-ci.yml

```diff
@@ -82,6 +82,11 @@ Build and push image:
       docker build \
       --build-arg "NPM_REGISTRY=https://nexus.osdc.io/repository/npm-all/" \
```

Comment:
Would it be too hard to get version and bake it as label?

https://github.com/NCI-GDC/gdcapi/blob/4dac2b9c63c746e17704c9633bf8d8cce0623c8c/docker.sh#L33-L37

Assuming it's simple to get version in a node app.

Reply:
I thought we decided to remove all versions from labels because we are not rebuilding for tags, 
and the versions will just confuse.